ohapi/cassettes/test_upload_file_remote_info_not_none_matching_file_size.yaml

```diff
@@ -0,0 +1,37 @@
+interactions:
+- request:
+    body: null
+    headers:
+      Accept: ['*/*']
+      Accept-Encoding: ['gzip, deflate']
+      Connection: [keep-alive]
+      User-Agent: [python-requests/2.9.1]
+    method: GET
+    uri: https://stackoverflow.com/questions/33184342/how-do-i-mock-an-open-write-without-getting-a-no-such-file-or-directory
```

Comment:
Should the `uri` here be a SO question?

Reply:
I think I missed this one. Thanks for pointing it out :)